recipes/openscenegraph/all/conanfile.py

```diff
@@ -41,6 +41,7 @@ class OpenSceneGraphConanFile(ConanFile):
         "with_png": [True, False],
         "with_tiff": [True, False],
         "with_zlib": [True, False],
+        "opengl_profile": ["GL1", "GL2", "GL3", "GLCORE", "GLES1", "GLES2", "GLES3", "GLES2+GLES3"],
```

Comment:
Use lower-case as preferred to keep the pattern, so you can use string.toupper() when passing to the cmake definition.